target/linux/ath79/base-files/etc/board.d/02_network

```diff
@@ -13,6 +13,7 @@ ath79_setup_interfaces()
 		ucidef_set_interface_lan "eth0" "dhcp"
 		;;
 	"avm,fritz300e"|\
+	"ligowawe,dlb5-propeller"|\
```

Comment:
Quotes have been removed in this file. Maybe apply to that.
Also consider this for the uboot-envtools switch.